### PR TITLE
REQ-03: refactor code, ask whether to add pet

### PR DESCRIPTION
**Pull request by KaterynaMir**
The code was refactored. The ability to add a pet was moved to the ClientService class.

### CODE REVIEW

**Comment by oleksandr-sydorskyi** [comment]
It's a great solution to put the functionality of adding a pet into a separate method. The only thing that made me doubt in which class this method should be placed in ClientService or PetService? The responsibility of the ClientService class typically includes operations related to managing clients, such as registering new clients, updating client information, and handling client-related actions. Seems, that adding a pet for a client fits well within the scope of client management operations.

File: src/main/java/com/magicvet/service/ClientService.java [hunk 1]
```diff
@@ -29,6 +30,20 @@ public Client registerNewClient(){
...
+        if (answerAddPet.toLowerCase().trim().startsWith("y")) {
+            System.out.println("Adding a new pet.");
+            Pet pet = petService.registerNewPet();
+            client.setPet(pet);
+            pet.setOwnerName(client.getFirstName() + " " + client.getLastName());
+            System.out.println("Pet has been added.");
+        } else {
+            System.out.println("Have a nice day!");
+        }
+    }
```
Comment (oleksandr-sydorskyi) [inline src/main/java/com/magicvet/service/ClientService.java]: What happens if the client inputs wrong data except y or n? The current implementation of the addPet method does not handle it. Let's enhance the method to handle invalid input by repeatedly prompting the user until valid input is provided. You can achieve this by using a loop.

Comment (KaterynaMir) [reply src/main/java/com/magicvet/service/ClientService.java]: According to the reviewer's suggestion, the check of the client's input was added to the addPet method.

Comment (oleksandr-sydorskyi) [reply src/main/java/com/magicvet/service/ClientService.java]: Everything looks good! You can merge it into the main branch.

**Pull request merged**